examples/tests/valid/util.env.w

```diff
@@ -1,10 +1,15 @@
+/*\
+env:
+  MY_VAR: my value
+\*/
```

Comment:
Makes sense. @MarkMcCulloh what would be the procedure to manually add valid/invalid TS test files (where do I put them and how do I name them) to get them working with our current test framework and snapshots and all that?
I'm also fine with just noting this for the future and keeping the `env` feature, which is pretty useful and clear and has a simple implementation.

Reply:
Might be worth a sync meeting. I'm open to making our e2e tests more explicit rather than having this system to generate tests, but I also feel like the current system makes tests pretty easy to make. I'm on the fence though, especially after the recent discussion about doing less snapshotting.

In any case, I'd vote that any changes like that should be in a separate PR.